pkg/operator/starter.go

```diff
@@ -58,20 +64,28 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 	kubeInformersForNamespaces := v1helpers.NewKubeInformersForNamespaces(kubeClient,
 		"",
 		"kube-system",
-		"cert-manager",
```

Comment:
We want this informer as well, to enforce resources created in `cert-manager` namespace.

Reply:
`operatorclient.TargetNamespace` already has the same value "cert-manager", as the value was duplicated I ended up removing the former.